iam.tf

```diff
@@ -55,22 +59,32 @@ resource "aws_iam_role_policy" "codebuild_policy" {
           "ssm:GetParameters",
           "ssm:GetParametersByPath"
         ],
-        Resource = "*"
+        Resource = "arn:aws:ssm:${var.region}:${data.aws_caller_identity.current.account_id}:parameter/${var.parameter_name}"
       },
       {
         Effect   = "Allow",
         Action   = [
           "ecr:BatchGetImage",
           "ecr:GetDownloadUrlForLayer"
         ],
-        Resource = "*"
+        Resource = "arn:aws:ecr:${var.region}:${data.aws_caller_identity.current.account_id}:repository/${var.environment}-${var.pipeline_name}-codebuild*"
```

Comment:
Where is that repository created?

Reply:
It depends. The template doesn't create the repository, it assumes that you already have an ECR repository and only requires you to provide its URI. Should I add the creation of the ECR repository to the template?

Reply:
I believe so, yes